.travis.yml

```diff
@@ -1,33 +1,26 @@
+# See https://github.com/silverstripe/silverstripe-travis-support for setup details
+
+sudo: false
+
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
   - 5.6
+  - 7.0
+  - 7.1
 
 env:
-  - DB=MYSQL CORE_RELEASE=3.4
-
-matrix:
-  include:
-    - php: 7.1
-      env: DB=MYSQL CORE_RELEASE=3.6 COVERAGE="--coverage-clover=coverage.xml"
-    - php: 7.1
-      env: DB=PGSQL CORE_RELEASE=3
+  global:
+    - DB=MYSQL CORE_RELEASE=4
 
 before_script:
   - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
   - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
   - cd ~/builds/ss
 
 script:
-  - vendor/bin/phpunit "$COVERAGE" ideannotator/tests
+  - vendor/bin/phpunit ideannotator/tests
 
-# Share coverage results with codecov.io
 after_success:
-  - >
-    test "$COVERAGE" != ""
-    && mv coverage.xml ~/build/$TRAVIS_REPO_SLUG
-    && cd ~/build/$TRAVIS_REPO_SLUG
-    && bash <(curl -s https://codecov.io/bash)
+  - cd ~/builds/ss/ideannotator
```

Comment:
I'm not opposed to switching to CircleCI, but think if it's done it should be done for all SL repos. The reason we use Travis is because SilverStripe uses it, so we follow a sort of example/consistency.

In this case you can't use the travis-support module for SS4 modules any more - if you take a look at some of the other SilverStripe supported modules (e.g. siteconfig, reports, etc - one without Behat is simpler) you'll see the new style of standalone Travis configuration which is basically just installing the core SilverStripe recipe and running the tests as per normal.

The default SS4 Travis configuration also includes PHPCS validation which replaces SilverLeague's previous template of enforcing this in Scrutinizer CI (not in this repo yet)

Reply:
I'm not planning on moving it to CircleCI, I just switched it because Travis refused to run (for now), and I didn't want to be bothered by fixing Travis as setting up CircleCI (for now) was faster/easier.

Reply:
(I'm way more comfortable setting up CircleCI than Travis, heh ;) )